doc/_includes/tutorials/common/ncbi-export.md

```diff
@@ -9,26 +9,34 @@ You will be forwarded to a page where you must enter metadata about the uploaded
 * BioProject ID - BioProject to submit files to. This project must be created in NCBI prior to this submission.
 * Organization - Name of organization submitting these samples.
 * Identifier Namespace - Prefix to use for submission identifiers in NCBI.  This prefix will be used to assign upload identifiers in the SRA but may not be visible in the uploaded files.
-* Release Date - Submission will not be made public until after the chosen release date.
+* Release Date - Submission won't be public until after the chosen release date.
 
 ![NCBI project metadata]({{ site.baseurl }}/images/tutorials/common/ncbi-export/ncbi-project-metadata.png)
 
-Next you must fill in information about the samples to be uploaded.  For more detailed information about these fields see [NCBI's SRA Handbook](http://www.ncbi.nlm.nih.gov/books/NBK47528/) ([Library Information](http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._library_descriptor_te/), [Sequencing Platform Description](http://www.ncbi.nlm.nih.gov/books/NBK54984/table/SRA_Glossary_BK.T._platform_descriptor_t/)).
+Next you must fill in information about the samples to be uploaded.
 
 * BioSample ID - NCBI BioSample to add files to. This sample must be created in NCBI prior to this submission.
-* Library Name - The submitter's name for this library.
+* Library Name - The submitters name for this library.
 * Library Strategy - Sequencing technique intended for this library.
 * Library Source - The type of source material that is being sequenced.
 * Library Construction Protocol - Free form text describing the protocol by which the sequencing library was constructed.
 * Instrument Model - The sequencing platform used to produce the data.
 * Library Selection - Whether any method was used to select for or against, enrich, or screen the material being sequenced.
 
+Default values fro all samples can be set by clicking on the "Default Sample Settings" and setting the values for Library Strategy, Library Source, Library Construction Protocol, Instrument Model, Library Selection.  You can set any number of these fields that you want, and the values can be overwritten directly within each sample.
```

Comment:
Typo in `for`

Reply:
Updated in 40a8fe83aa57a070209f2e26979a5933a89bbfe3